test/test_core.py

```diff
@@ -1002,6 +1004,7 @@ def test_longjmp2(self):
 
   @needs_dylink
   @with_all_sjlj
+  @no_js_math('TODO: https://github.com/emscripten-core/emscripten/issues/25162')
```

Comment:
`JS_MATH is not compatible with MODULE_MODULE=1`.

Can you also add an error to link.py if these are used together?

Reply:
Updated.